consensus/src/round_manager.rs

```diff
@@ -330,9 +331,17 @@ impl RoundManager {
         new_round_event: NewRoundEvent,
     ) -> anyhow::Result<ProposalMsg> {
         // Proposal generator will ensure that at most one proposal is generated per round
+        let sync_info = self.block_store.sync_info();
+        let mut sender = self.network.clone();
+        let callback = async move {
+            sender
+                .broadcast(ConsensusMsg::SyncInfo(Box::new(sync_info)))
+                .await;
+        }
+        .boxed();
```

Comment:
So if I understand this PR right, you broadcast the SyncInfo over `DirectSend`.  Now that you're broadcasting more, this is supposed to slow everything down?

Did you stop broadcasting the `SyncInfo` elsewhere?

Reply:
the broadcast is only triggered when we're waiting for mempool and only triggered once per pull. I think it's fine when the system is idle.